NEMP/mods.json

```diff
@@ -4561,5 +4561,14 @@
             "promo": "recommended",
             "promoType": "version"
         }
+    },
+    "Brain Stone Mod": {
```

Comment:
This needs to be the exact name your mod has in Not Enough Mods (`BrainStoneMod`).